app.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const app = express();
-const { getTopics, getEndPoints, getArticleID, articleCommentCount } = require('./controllers/controller')
+const { getTopics, getEndPoints, getArticleID, articleCommentCount, getArticleComments} = require('./controllers/controller')
 
 app.use(express.json());
```

Comment:
We don't need express.json() until we are working on a post or a patch so please remove 👍